GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -462,6 +463,11 @@ void onDrawEyeView(int eye, float fovDegrees) {
         if (!(mSensoredScene == null || !mMainScene.equals(mSensoredScene))) {
             GVRCameraRig mainCameraRig = mMainScene.getMainCameraRig();
 
+            if(eye == 0) {
```

Comment:
Isn’t this if(eye == 0) statement is redundant, as it will go to the else statement below? Eye will be either 0 or 1. Also, it assumes it is called 0 first.